examples/pytorch/torchscript/IrisClassification/MLproject

```diff
@@ -0,0 +1,15 @@
+name: iris-classification
+
+conda_env: conda.yaml
+
+entry_points:
+  main:
+    parameters:
+      epochs: {type: int, default: 50}
+      tracking_uri: {type: str, default: 'http://localhost:5000'}
```

Comment:
```suggestion
```

Reply:
removed tracking uri argument and updated readme files with necessary details